Dockerfile

```diff
@@ -31,6 +31,12 @@ RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH GO111MODULE=on go build -a -
 # Extend Grist image.
 FROM $BASE as merge
 
+# Enable sandboxing by default. It is generally important when sharing with
+# others. You may override it, e.g. "unsandboxed" uses no sandboxing but is
+# only OK if you trust all users fully.
+ENV GRIST_SANDBOX_FLAVOR=gvisor
+ENV GVISOR_FLAGS="-unprivileged -ignore-cgroups"
```

Comment:
I'm a bit uncomfortable spreading around these flags to other repositories. They are kind of an implementation detail, and could change if there is progress on https://github.com/google/gvisor/issues/4371 (and I see that avagin recently assigned themselves this task). Would they not be inherited from the grist-core image? Do you want to set them here to be confident they are set?